.gitignore

```diff
@@ -0,0 +1,11 @@
+# Node files
+node_modules
+.serverless
+.build
+dist
+
+# MacOS files
+.DS_Store
```

Comment:
One of these files got checked into the `__tests__` folder.

Reply:
Thank you for catching that! Removed in a117ddfb6e2ecedfc19edefda7a76b42375cd147